src/components/dashboard/CategoryTabbar.tsx

```
@@ -0,0 +1,89 @@
+import React, {useEffect} from 'react';
+import {useTheme} from '../../context/theme/useTheme';
+import CustomText from '../common/CustomText';
+import {FlatList, StyleSheet, TouchableOpacity, View} from 'react-native';
+import {Theme} from '../../constants/Types';
+import categoryService from '../../appwrite/category';
+import {Category} from '../../appwrite/types/category';
+
+interface CategoryTabbarProps {
+  onCategoryPress: (category: Category) => void | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)